src/Common/src/System/Text/OSEncoding.Windows.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Text
     internal sealed class OSEncoding : Encoding
     {
         private readonly int _codePage;
-        private string _encodingName;
+        private string? _encodingName;
```

Comment:
is this supposed to be annotated with this project?

Reply:
good point. I missed that this was not inside the project... fixing

Reply:
This is also used for System.Diagnostics.Process. 
I could be just adding `#nullable enable` to this file to complete work for System.Console. Pushing that change now